.github/workflows/usermods.yml

```
@@ -0,0 +1,71 @@
+  push:
+    paths:
+      - usermods/**
+      - .github/workflows/usermods.yml
+    
+jobs:
+
+  get_usermod_envs:
+    name: Gather Usermods
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)